apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/index.js

```diff
@@ -14,10 +14,10 @@ import edit from './edit';
 import './style.scss';
 import './site-logo';
 
-if ( 'wp_template' !== fullSiteEditing.editorPostType ) {
+if ( 'wp_template_part' !== fullSiteEditing.editorPostType ) {
 	registerBlockType( 'a8c/template', {
-		title: __( 'Template' ),
-		description: __( 'Display a template.' ),
+		title: __( 'Template part' ),
+		description: __( 'Display a template part.' ),
```

Comment:
capitalPdangit, like the ones in register_template_post_types?

Reply:
Thanks! Updated in 4f72c95180f56f248285773e642cf87248bee870.